packages/unmock-core/src/service/state/utils.ts

```diff
@@ -26,6 +26,22 @@ interface ICodesToMediaTypes {
 
 const debugLog = debug("unmock:state:utils");
 
+/**
+ * Works like array.some(), except it exhauses all array elements before returning.
+ * @param arr
+ * @param callbackfn
+ */
+export const anyFn = <T>(
```

Comment:
I would not expect the internal implementation of `anyFn` to affect the behaviour unless the `callbackFn` was doing side-effects behind the scenes. I guess this function is made for applying side-effects like that? I think it might be clearer to apply side-effects in a `forEach` or `applyAll` rather than `anyFn` 🤔